css/style.css

```diff
@@ -0,0 +1,67 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  background-color: skyblue;
+  text-align: center;
+  /* display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  position: absolute; */
```

Comment:
Please do not leave unused codes as a comments in your codebase. Unused codes left as comments make your codebase unclean.

Reply:
Great job here 👌